src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -77,7 +86,8 @@ private Loan(JsonObject representation, Item item, User user, User proxy,
 
     this.originalDueDate = originalDueDate == null ? getDueDate() : originalDueDate;
 
-    this.loanPolicy = loanPolicy;
+    this.overdueFinePolicyId = getProperty(representation, OVERDUE_FINE_POLICY_ID);
```

Comment:
why _loanPolicy_ was removed and _lostItemPolicyId_, _overdueFinePolicyId_ fields were added?

Reply:
1) loanPolicy was removed because the loanPolicy filed is moved to Polices class. Policies field is assigned in the first line in the constructor of the Loan.
2)  lostItemPolicyId, overdueFinePolicyId were added in order to avoid using json representation. (As we did for checkinServicePoint and checkoutServicePoint)